server/main.lua

```diff
@@ -39,12 +39,37 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleOwner = GetVehicleOwner(plate)
+		local bolo, title, boloId = GetBoloStatus(plate, title, boloId)
+		--local warrant, owner, incidentId = GetWarrantStatus(plate, owner, incidentId)
+		local warrant, owner, incidentId = GetWarrantStatus(plate)
+		local driversLicense = PlayerData.metadata['licences'].driver
+		local driverunlicensed = nil
+
 		if bolo == true then
-			TriggerClientEvent("wk:togglePlateLock", src, cam, true, bolo)
+			TriggerClientEvent('QBCore:Notify', src, 'BOLO ID: '..boloId..' | Title: '..title..' | Registered Owner: '..vehicleOwner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+		if warrant == true then
+			TriggerClientEvent('QBCore:Notify', src, 'WANTED - INCIDENT ID: '..incidentId..' | Registered Owner: '..owner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+
+		if driversLicense == false then
+			driverunlicensed = true
+			if driverunlicensed == true then

```

Comment:
This if condition will always result in true, since you are hardcoding `driverunlicensed` to true right above it. As such there is never a falsy path that could be taken with this conditional, so its not needed. 

You can remove lines 58 and 59, the check for driverunlicensed is not necessary at this point. Your check on line 57 `if driversLicense == false then` does the check for you already.

Reply:
Amended